packages/editor/src/components/sidebar/post-summary.js

```diff
@@ -87,7 +88,12 @@ export default function PostSummary( { onActionPerformed } ) {
 									</VStack>
 									<PostStickyPanel />
 									<PostFormatPanel />
-									{ isTemplate && <TemplateAreas /> }
+									{ isTemplate && (
```

Comment:
It would be cool to avoid this adhoc checks here and move them to the components for now.

Reply:
I'm not too strongly opinionated though :)